### DISCUSSION TITLE
Getting response

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#4696] [Q&A]
How can I get the response (ack) from the message sent out a destination? I tried putting this in the Response transformer:
var myResponse = responseMap.get(0);
logger.info('response ' + myResponse);

but it only displays null.  I can see an hl7 message in the response when I look at the message going out.  I need to get that response and send it out another destination.

**Comment by jonbartels** [comment]
https://www.nextgen.com/-/media/files/nextgen-connect/nextgen-connect-311-user-guide.pdf page 258 explains this better than I can. That lists all the special objects and their types that you can use in a response transformer.

**Reply by rutledj** [reply]
Thanks. That was helpful.
